go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -36,12 +39,14 @@ type egressFirewallRule struct {
 
 type destination struct {
 	cidrSelector string
+	dnsName      string
 }
 
 func newEgressFirewall(egressFirewallPolicy *egressfirewallapi.EgressFirewall) *egressFirewall {
 	ef := &egressFirewall{
 		name:        egressFirewallPolicy.Name,
 		namespace:   egressFirewallPolicy.Namespace,
+		uid:         egressFirewallPolicy.UID,
```

Comment:
where is this uid used later on and what for? Mybe it's due to GH folding but I don't see any calls to it afterwards.

Reply:
The openshift code uses the UID and I thought I was going to need it but I don't use it you are right. I removed it